.drone.star

```diff
@@ -2407,9 +2406,6 @@ def runWebuiAcceptanceTests(ctx, suite, alternateSuiteName, filterTags, extraEnv
         }, {
             "name": "configs",
             "path": "/srv/config",
-        }, {
-            "name": "uploads",
-            "path": "/usr/src/app/filesForUpload",
```

Comment:
middleware client (here `web`) does not really need to know about the uploads path